src/mutation_data.cpp

```diff
@@ -411,6 +411,15 @@ void mutation_branch::load( const JsonObject &jo, const std::string & )
         spells_learned.emplace( sp, ja.next_int() );
     }
 
+    for( JsonArray ja : jo.get_array( "craft_skill_bonus" ) ) {
+        const skill_id skid( ja.next_string() );
+        if( skid.is_valid() ) {
```

Comment:
What if the skill definition is loaded *after* the mutation is loaded? This will report that skill id as invalid. Id checking should be done after everything has been loaded.